Assets/Mirror/Runtime/Messages.cs

```diff
@@ -363,7 +329,11 @@ public void Deserialize(NetworkReader reader)
         {
             netId = reader.ReadPackedUInt32();
             isLocalPlayer = reader.ReadBoolean();
-            sceneId = reader.ReadUInt64();
+            sceneId = reader.ReadPackedUInt64();
+            if (sceneId != 0)
```

Comment:
why != 0 here?
shouldn't assetid be used when not a scene object?

Reply:
fixed